stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/internal/DateDividerViewHolder.kt

```diff
@@ -40,6 +40,8 @@ internal class DateDividerViewHolder(
     ),
 ) : DecoratedBaseMessageItemViewHolder<MessageListItem.DateSeparatorItem>(binding.root, decorators) {
 
+    private val defaultCornerRadius = DEFAULT_CORNER_RADIUS.dpToPxPrecise()
```

Comment:
Same here, the memory size will be increase because `DateDividerViewHolder` is used multiple times on our SDK